### PR TITLE
Added new iam user login and group

### PR DESCRIPTION
**Pull request by shawngerrard**
I've scaled back some updates so we can get some incremental progress now that I know a little more about securing aws organizations and working with aws iam users, roles, and policies.

This update is to add a new _admins_ group into the master iam account, as well as a new user (with login profile) who will be attached to this group. This sets the foundations for creating the credentials to allow users to administer asterion resources, and then applying access policies to these users. 

**Note:** The password exported is received by aws in plain-text. At this stage, the user currently has no authorization to use or read asterion resources by default and can therefore only log in to the asterion account. This _must_ be updated to using pgp public keys once policies are applied that will allow this user access to asterion resources.

### CODE REVIEW

File: infra-aws/__main__.py [hunk 1]
```diff
@@ -23,7 +24,8 @@
 	cidr_block="10.0.101.0/24",
     map_public_ip_on_launch=True,
 	tags={ "Name": "asterion-infra-ec2" },
-	vpc_id=vpc.id
+	vpc_id=vpc.id__(self, name):
+        self.na
```
Comment (jmhbnz) [inline infra-aws/__main__.py]: Curious, what is `self.na` being used here?

File: org-asterion/asterion-aws-org-components.dot [hunk 2]
```diff
@@ -0,0 +1,58 @@
+digraph G {
```
Comment (jmhbnz) [inline org-asterion/asterion-aws-org-components.dot:1]: Love the use of grahpviz for diagrams as code! ❤️

**Review by jmhbnz — APPROVED** [review]

File: infra-aws/__main__.py (hunk 1 above)
Comment (shawngerrard) [reply infra-aws/__main__.py]: That's not meant to be there! 😅 Looks like I managed to transpose something erroneously here and missed the diff. I've corrected it now.

**Comment by jmhbnz** [comment]
@shawngerrard I see build is failing, unsure if that is expected but from my perspective merge is approved so hit merge whenever you're ready 👍🏻

**Comment by shawngerrard** [comment]
> @shawngerrard I see build is failing, unsure if that is expected but from my perspective merge is approved so hit merge whenever you're ready 👍🏻

For now it's expected (it's always failed on this repo), but it'll be fixed shortly. :)

**Pull request merged**